mne/annotations.py

```diff
@@ -567,6 +567,49 @@ def crop(self, tmin=None, tmax=None, emit_warning=False, verbose=None):
 
         return self
 
+    @verbose
+    def set_durations(self, mapping, verbose=None):
+        """Set annotation durations(s). Operates inplace.
+
+        Parameters
+        ----------
+        mapping : dict | float | int
```

Comment:
```suggestion
        mapping : dict | float
```
I would say float only as it's seconds